pytorch_lightning/trainer/trainer.py

```diff
@@ -62,13 +64,15 @@ class Trainer(TrainerIOMixin,
               TrainerEvaluationLoopMixin,
               TrainerTrainLoopMixin,
               TrainerCallbackConfigMixin,
+              TrainerCallbackHookMixin
               ):
 
     def __init__(
             self,
             logger: Union[LightningLoggerBase, bool] = True,
             checkpoint_callback: Union[ModelCheckpoint, bool] = True,
             early_stop_callback: Optional[Union[EarlyStopping, bool]] = None,
+            callbacks: list = [],
```

Comment:
I recently added typehints to the Trainer and would like to suggest to make the type List[Callback]